packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -70,49 +73,59 @@ export const DefaultWithChildren: Story = {
   },
 };
 
-export const CardLoading: Story = {
```

Comment:
I removed this story because there is already Card Loading story in Card stories and also it's not necessary to put Loading component when Card has already loading prop set.